### ISSUE TITLE
False "Cargo.lock is present in your .gitignore and is also committed."

### ISSUE THREAD

**Issue by nxsaken** [MarcoIeni/release-plz#1608]
## Bug description

I'm getting an error when using `release-plz update`:

```
Cargo.lock is present in your .gitignore and is also committed. Remove it from your repository or from your `.gitignore` file.
```

However, I never committed `Cargo.lock`. The initial commit added it to `.gitignore`.

## To Reproduce

1. create a new project (`cargo new foo --lib`)
2. add Cargo.lock to .gitignore
3. stage and commit `.gitignore`, `Cargo.toml`, and `lib.rs`
4. run `release-plz update`

## Environment

- OS: macOS 14.5
- release-plz version: 0.3.83, checked main branch too

**Comment by MarcoIeni** [comment]
Ok, any reproduction steps? How can I debug this?

**Comment by nxsaken** [comment]
Just checked with a new project:
- create a new project (`cargo new foo --lib`)
- add Cargo.lock to .gitignore
- stage and commit `.gitignore`, `Cargo.toml`, and `lib.rs`
- run `release-plz update`

**Comment by MarcoIeni** [comment]
Sorry if it took me so long to look into this!

You were right. I fixed this in #1662 👍
